plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -1446,7 +1480,7 @@ $.extend(DataTable.prototype, UIControl.prototype, {
         var self = this;
 
         // highlight all columns on hover
-        $(domElem).on('mouseenter', 'td', function (e) {
+        $(domElem).on('mouseenter', 'td:not(.cellSubDataTable)', function (e) {
```

Comment:
There was a weird flickering effect when moving the mouse into a subtable sometimes. This seems to have been caused by this event, which isn't needed on a td, that contains a subtable.